src/controllers/post.js

```diff
@@ -61,3 +61,14 @@ export const getAll = async (req, res) => {
     return sendDataResponse(res, 400, { err: err.message })
   }
 }
+
+export const deletePost = async (req, res) => {
+  const postId = +req.params.id
+  if (!postId) throw new Error('The ID you have provided is incorrect')
```

Comment:
Catch this error - moving into the try in this function, or adding it in the parent function